tests/conftest.py

```diff
@@ -11,11 +16,22 @@ class PyTestUtils:
     class PillTest:
         pass
 
-try:
-    from pytest_terraform.tf import LazyReplay
-    LazyReplay.value = True
-except ImportError: # noqa
-    pass
+    class LazyReplay:
+        pass
+
+    class LazyPluginCacheDir:
+        pass
+
+
+# If we have C7N_FUNCTIONAL make sure Replay is False otherwise enable Replay
+LazyReplay.value = not strtobool(os.environ.get('C7N_FUNCTIONAL', 'no'))
+LazyPluginCacheDir.value = '../.tfcache'
+
+
+def pytest_configure(config):
+    # Only register pytest-terraform hooks if the plugin is available
+    if config.pluginmanager.hasplugin("terraform"):
+        config.pluginmanager.register(TerraformHooks())
```

Comment:
please rename these to TerraformRewriteHooks to denote their purpose a bit more clearly.